public/schema.json

```diff
@@ -1818,6 +1818,15 @@
             "type": "string",
             "description": "The error message",
             "example": "Bad request. Missing or invalid input parameters."
+          },
+          "fields": {
+            "description": "The error fields",
+            "type": "object",
```

Comment:
Can't just have object, needs some sort of properties defined. If properties unknown as in this case would need to configure it:
- https://stackoverflow.com/questions/32044761/json-schema-with-unknown-property-names
- https://json-schema.org/understanding-json-schema/reference/object#patternProperties